<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSettings</class>
 <widget class="QWidget" name="DlgSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>238</red>
        <green>238</green>
        <blue>236</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>238</red>
        <green>238</green>
        <blue>236</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>190</red>
        <green>190</green>
        <blue>190</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DlgSettings{
	
	background-image: url(:/ui/background_opacity.png);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layouthead">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lb_username">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>User Name</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#btn_close{
background-color: transparent;
background: none;
border-style: outset;
border-width: 2px;
border-color: beige;
background-repeat: none;
}
#btn_close:pressed
{
	background-color: rgba(252, 3, 3, 155);
	border-color: rgb(186, 189, 182);
}</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
     <item>
      <widget class="QLabel" name="labelToken">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Lichess token:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_inputtoken">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="PlaceholderText">
           <brush brushstyle="SolidPattern">
            <color alpha="128">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="PlaceholderText">
           <brush brushstyle="SolidPattern">
            <color alpha="128">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="PlaceholderText">
           <brush brushstyle="SolidPattern">
            <color alpha="128">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#btn_inputtoken{
background-color: transparent;
background: none;
border-style: outset;
border-width: 2px;
border-color: beige;
background-repeat: none;
}
#btn_inputtoken:pressed
{
	background-color: rgba(252, 3, 3, 155);
	border-color: rgb(186, 189, 182);
}</string>
       </property>
       <property name="text">
        <string>input token ...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4,1,1">
     <item>
      <widget class="QLabel" name="labelWifi">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wifi:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSpeedMode_3">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Dongvua990</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_wifi_scan">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>35</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#btn_wifi_scan{
background-color: transparent;
background: none;
border-style: outset;
border-width: 2px;
border-color: beige;
background-repeat: none;
}
#btn_wifi_scan:pressed
{
	background-color: rgba(252, 3, 3, 155);
	border-color: rgb(186, 189, 182);
}</string>
       </property>
       <property name="text">
        <string>scan</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,1,2">
     <item>
      <widget class="QLabel" name="labelSpeedMode">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>High Speed Move</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_speedMode">
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>34</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#btn_speedMode{
	
	border-image: url(:/ui/toggle_off.png);
}
#btn_speedMode:checked{
	border-image: url(:/ui/toggle_on.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,1,1,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sound</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_sound">
       <property name="minimumSize">
        <size>
         <width>69</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>34</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#btn_sound{
	
	border-image: url(:/ui/toggle_off.png);
}
#btn_sound:checked{
	border-image: url(:/ui/toggle_on.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_help">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>35</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>238</red>
             <green>238</green>
             <blue>236</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#btn_help{
background-color: transparent;
background: none;
border-style: outset;
border-width: 2px;
border-color: beige;
background-repeat: none;
}
#btn_help:pressed
{
	background-color: rgba(252, 3, 3, 155);
	border-color: rgb(186, 189, 182);
}</string>
       </property>
       <property name="text">
        <string>help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
